### ISSUE TITLE
Show all live binding values

### ISSUE THREAD

**Issue by WangLarry** [mui/mui-toolpad#1000]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

To debug app, if can show all live binding values in a window, which will helpful, just like 'View DOM'

**Comment by Janpot** [comment]
Agreed, currently we already show live values in the binding editor global scope explorer. We can make this view accessible in its own panel as well. As a benchmark, retool recently released their devtools feature:


https://user-images.githubusercontent.com/2109932/190191534-e4f2e24e-52a2-4090-b90e-2abb3a87beab.mov

**Comment by JerryWu1234** [comment]
> Agreed, currently we already show live values in the binding editor global scope explorer. We can make this view accessible in its own panel as well. As a benchmark, retool recently released their devtools feature:
> 
>  Screen.Recording.2022-09-13.at.15.12.08.mov

May I ask how to open this scope explorer(dialog in the video)?

**Comment by Janpot** [comment]
> May I ask how to open this scope explorer(dialog in the video)?

go to [retool.com](https://retool.com/), and sign up for an account 😄

**Comment by JerryWu1234** [comment]
retool turns out to be a website. lol

**Comment by JerryWu1234** [comment]
thank you
